src/uniprotkb/components/entry/similar-proteins/SimilarProteins.tsx

```diff
@@ -82,73 +86,75 @@ const SimilarProteins: FC<{
-                        .join(' OR ')})`,
-                    }}
+        {clusterData ? (
+          <Tabs>
+            {Object.entries(uniRefEntryTypeToPercent).map(
+              ([clusterType, percentValue]) =>
+                clusterType in clusterData &&
+                clusterData[clusterType as UniRefEntryType] && (
+                  <Tab
+                    id={clusterType}
+                    title={`${percentValue} identity`}
+                    key={clusterType}
                   >
-                    View all
-                  </Button>
-                </Tab>
-              )
-          )}
-        </Tabs>
+                    {Object.entries(
+                      clusterData[clusterType as UniRefEntryType] || {}
+                    ).map(([representativeId, clusters]) => (
+                      <section key={representativeId} className="text-block">
+                        <h4>{representativeId}</h4>
+                        {clusters.map((row) => {
+                          const unirefEntryUrl = getEntryPath(
+                            Namespace.uniref,
+                            row.id
+                          );
+                          return (
+                            <section key={row.id}>
+                              <h5>
+                                <Link to={unirefEntryUrl}>{row.id}</Link>
+                              </h5>
+                              <SimilarProteinsTable members={row.members} />
+                              {row.memberCount - row.members.length - 1 > 0 && (
+                                <Link to={unirefEntryUrl}>
+                                  {row.memberCount - row.members.length - 1}{' '}
+                                  more
+                                </Link>
+                              )}
+                            </section>
+                          );
+                        })}
+                        <hr />
+                      </section>
+                    ))}
+                    {/* TODO: This query doesn't seem to work currently */}
+                    <Button
+                      element={Link}
+                      to={{
+                        pathname: LocationToPath[Location.UniProtKBResults],
+                        search: `query=(${data?.results
+                          .filter(({ entryType }) => entryType === clusterType)
+                          .map(
+                            ({ id }) =>
+                              `uniref_cluster_${clusterType.replace(
+                                'UniRef',
+                                ''
+                              )}:${id}`
+                          )
+                          .join(' OR ')})`,
+                      }}
+                    >
+                      View all
+                    </Button>
+                  </Tab>
+                )
+            )}
+          </Tabs>
+        ) : (
+          <em>No similar proteins found.</em>
```

Comment:
Not sure about the wording. If we arrived there, it's when we have UniParc, but no UniProt entry, right? I would specify this, something like "No similat UniProt entry found" or something like this.
I believe we should be displaying the UniParc entries too anyway, marked clearly as UniParc, but that's different from the current behaviour of the website though.

Reply:
I agree that in the long run we should also include UniParc. Will update the text to "No similar UniProtKB entry found" to make less ambiguous.